reference/orders.v2.oas2.yml

```diff
@@ -5189,6 +5222,29 @@ components:
           shipping_quotes:
             url: 'https://api.bigcommerce.com/stores/{$$.env.store_hash}/v2/orders/163/consignments/shipping/64/shipping_quotes'
             resource: /orders/163/consignments/shipping/64/shipping_quotes
+    shippingConsignment_Put:
+      allOf:
+        - type: object
+          properties:
+            id:
+              type: integer
+              example: 99
+              description: ID of shipping consignment to update.
+          required:
+            - id
+        - type: object
+          properties:
+            line_items:
+              type: array
+              minItems: 1
+              items:
+               type: array
+               items:
+                anyOf:
+                 - $ref: '#/components/schemas/orderCatalogProduct_Put'
+                 - $ref: '#/components/schemas/orderCatalogProduct_Post'
+                 - $ref: '#/components/schemas/orderCustomProduct_Post'
+                 - $ref: '#/components/schemas/orderCustomProduct_Put'
```

Comment:
do we also support updating other address properties like first_name, last_name, etc ... ?

Reply:
yes we do, I think my idea was to put only the required fields here.

Reply:
I think we would need to add all the fields that are supported in the request payload. Maybe we can have a `shippingConsignment_base` and the `shippingConsignment_Put` and `shippingConsignment_Post` can include it inside ?

Reply:
Utilised an already existing `shippingConsignment_base` [here](https://github.com/bigcommerce/api-specs/pull/1017/commits/9a59d94d16cfee5c3826c74aede05723fa464a26#r1012376991)